controller/api/apis.go

```diff
@@ -2157,15 +2157,20 @@ type RESTScanReportData struct {
 }
 
 type RESTScanReport struct {
-	Vuls      []*RESTVulnerability `json:"vulnerabilities"`
-	Modules   []*RESTScanModule    `json:"modules,omitempty"`
-	Checks    []*RESTBenchItem     `json:"checks,omitempty"`
-	Secrets   []*RESTScanSecret    `json:"secrets,omitempty"`
-	SetIDs    []*RESTScanSetIdPerm `json:"setid_perms,omitempty"`
-	Envs      []string             `json:"envs,omitempty"`
-	Labels    map[string]string    `json:"labels,omitempty"`
-	Cmds      []string             `json:"cmds,omitempty"`
-	Verifiers []string             `json:"verifiers,omitempty"`
+	Vuls          []*RESTVulnerability   `json:"vulnerabilities"`
+	Modules       []*RESTScanModule      `json:"modules,omitempty"`
+	Checks        []*RESTBenchItem       `json:"checks,omitempty"`
+	Secrets       []*RESTScanSecret      `json:"secrets,omitempty"`
+	SetIDs        []*RESTScanSetIdPerm   `json:"setid_perms,omitempty"`
+	Envs          []string               `json:"envs,omitempty"`
+	Labels        map[string]string      `json:"labels,omitempty"`
+	Cmds          []string               `json:"cmds,omitempty"`
+	SignatureInfo *RESTScanSignatureInfo `json:"signature_data"`
```

Comment:
it should be
SignatureInfo *RESTScanSignatureInfo `json:"signature_data,omitempty"`